tests/data/data.json

```diff
@@ -1921,7 +1924,337 @@
     },
     "location": {
       "$ref": "https://ils.rero.ch/api/locations/loc7"
-    }
+    },
+    "holdings_type": "monograph"
+  },
+  "pattern1": {
```

Comment:
Probably I would put this directly in the test if it is used only once. I would keep here only one or two most frequent case.